app/components/EventItem/index.tsx

```diff
@@ -78,16 +78,12 @@ const TimeStamp = ({ event }: TimeStampProps) => {
         <Icon
           name="calendar-number-outline"
           size={20}
-          style={{ cursor: 'pointer', marginRight: '10px' }}
+          style={{ marginRight: '10px' }}
```

Comment:
Could replace the inline styling with a `gap` prop on `Flex` if you'd like, but not super important